src/composables/useOnboardingChecklist.ts

```diff
@@ -0,0 +1,85 @@
+import { computed, ref, watch } from 'vue';
+import { useProfiles } from '@/composables/useProfiles';
+import { useWeb3 } from '@/composables/useWeb3';
+import { useI18n } from '@/composables/useI18n';
+import { useApolloQuery } from '@/composables/useApolloQuery';
+import { PROPOSALS_QUERY } from '@/helpers/queries';
+import { ACTIVITY_VOTES_QUERY } from '@/helpers/queries';
+import { useFollowSpace } from '@/composables/useFollowSpace';
+
+const { t } = useI18n();
+
+const { apolloQuery } = useApolloQuery();
+const { profiles } = useProfiles();
+const { web3Account } = useWeb3();
+const { followingSpaces } = useFollowSpace();
+
+const profile = computed(() => profiles.value[web3Account.value]);
+
+const hasSetPublicProfile = computed(() =>
+  profile.value?.name ? true : false
+);
+const hasFollowedSpace = computed(() => followingSpaces.value.length);
+const hasVoted = ref(false);
+const hasCreatedProposal = ref(false);
+
+const onboardingChecklist = ref([
```

Comment:
It could be hard for new users figuring out how to complete each list item. Maybe each item should be collapsable with additional links and information. @ManuBernard do you have a designer with who we could try some things out?

Reply:
I have followed the design provided by @bonustrack :
https://www.figma.com/file/GS2AS8G3YCYjeH1dFTFpIn/Snapshot---onboarding-section?node-id=0%3A1

Not sure how to move forward, maybe you should talk together and see if we want to improve this in this PR or create a new one later?

Reply:
Oh I see. Yeah I wasn't aware of this design sorry for the confusion :)